ouroboros-network/src/Ouroboros/Network/PeerSelection/State/EstablishedPeers.hs

```diff
@@ -162,13 +163,16 @@ member peeraddr = Map.member peeraddr . allPeers
 insert :: Ord peeraddr
        => peeraddr
        -> peerconn
-       -> Time
+       -> Maybe Time
```

Comment:
Could you add a haddock; it's important to use this argument right.

Reply:
:+1: Will do.